packages/renderer/src/lib/ui/DetailsPage.svelte

```diff
@@ -39,7 +40,7 @@ function handleKeydown(e: KeyboardEvent) {
         </div>
         <div class="flex flex-col grow pr-2">
           <div class="flex flex-row">
-            <h1 aria-label="{title}" class="text-xl leading-tight">{title}</h1>
+            <div><CopyToClipboard title="{title}" clipboardData="{title}" class="text-xl" /></div>
```

Comment:
Then update classes for button to
```suggestion
            <div><CopyToClipboard title="{title}" clipboardData="{title}" class="text-xl inline-block mt-1" /></div>
```

That change wont affect style of the copy button in resources page and fix `Details page` layout

![image](https://github.com/containers/podman-desktop/assets/620330/774ad5a0-5e95-4ded-b65c-0cf975efd79b)